common/init.common.rc

```diff
@@ -26,16 +31,35 @@ on early-init
 
     setprop ro.hardware.audio.primary plus2e
 
+    start watchdogd
+
 on fs
-    mount_all /vendor/etc/fstab.orangepi_plus2e --early
+    mount_all /vendor/etc/fstab.${ro.hardware} --early
+    setprop ro.crypto.fuse_sdcard false
 
 on late-fs
-    mount_all /vendor/etc/fstab.orangepi_plus2e --late
+    mount_all /vendor/etc/fstab.${ro.hardware} --late
 
 on post-fs
     # set RLIMIT_MEMLOCK to 8MB
     setrlimit 8 8388608 8388608
 
+    # mount debugfs
+    mount debugfs /sys/kernel/debug /sys/kernel/debug mode=755
+
+    # disable transparent huge pages
+    write /sys/kernel/mm/transparent_hugepage/enabled "never"
+
+    chmod 0666 /dev/ion
+    chmod 0666 /dev/graphics/fb0
```

Comment:
Move to ueventd.rc?

Reply:
They are migrated from HiKey init.common.rc
Although moving them to ueventd.rc sounds resonable, it can be another refactor.

Reply:
I'll try to fragment this commit it into smaller chunks with specifying the motivation of each change.
Other 2 commits LGTM.